ledger-core/virtual/integration/contract_calls_test.go

```diff
@@ -77,6 +77,159 @@ func assertVCallRequest(t *testing.T,
 
 }
 
+func Test_NoDeadLock_WhenOutgoingComeToSameNode(t *testing.T) {
+	t.Skip("https://insolar.atlassian.net/browse/PLAT-432")
```

Comment:
no t.Log with link to test case in TR.

Reply:
I see cases logged below, this skip should be placed after logging test case.

Reply:
done